concordium-consensus/src/Concordium/KonsensusV1/SkovMonad.hs

```diff
@@ -650,7 +675,80 @@ migrateSkovFromConsensusV0 regenesis migration gsConfig@GlobalStateConfig{..} ol
                     },
                   SkovV1State
                     { _v1sSkovData = initSkovData,
-                      _v1sTimer = Nothing
+                      _v1sTimer = Nothing,
+                      _notifiedUnsupportedProtocol = False
+                    }
+                )
+    initWithBlockState `onException` liftIO (closeBlobStore pbscBlobStore)
+
+migrateSkovFromConsensusV1 ::
```

Comment:
Is this used anywhere in this PR?

Reply:
Can you please provide some documentation as to what it is doing in migration, for example carrying over the timeout as the current base timeout, rather than "current timeout".

Reply:
No, it is not used anywhere in this PR. It will be used in near-future PR though. Maybe I should remove it from this PR?

Reply:
I'm fine either way.

Reply:
Removed. (Will re-introduce in the follow-up PR.)